db/migrate/20170124121514_remove_bought_from_order_items.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveBoughtFromOrderItems < ActiveRecord::Migration[5.0]
+  def change
+    remove_column :order_items, :bought, :boolean
```

Comment:
Okay so if you want to remove a migration during development you can use the `delete`  generator, drop and recreate your db.
```
raid delete migration AddBoughtToOrderItems
```